x-pack/plugins/ml/public/application/jobs/jobs_list/components/utils.js

```diff
@@ -220,9 +220,22 @@ export async function cloneJob(jobId) {
     ]);
 
     const dataViewNames = await getDataViewNames();
-    const jobIndicesAvailable = dataViewNames.includes(datafeed.indices.join(','));
+    const dataViewTitle = datafeed.indices.join(',');
+    const jobIndicesAvailable = dataViewNames.includes(dataViewTitle);
 
     if (jobIndicesAvailable === false) {
+      const warningText = i18n.translate(
+        'xpack.ml.jobsList.managementActions.noSourceDataViewForClone',
+        {
+          defaultMessage:
+            'Unable to clone the anomaly detection job {jobId}. No data view exists for index {dataViewTitle}.',
+          values: { jobId, dataViewTitle },
+        }
+      );
+      getToastNotifications().addDanger({
```

Comment:
should we use `getToastNotificationService` instead?

Reply:
Switched this one to `getToastNotificationService`  in https://github.com/elastic/kibana/pull/117993/commits/b67a413b1d958a3c05428f18fa6218320ebd8a11.